tensorflow/core/kernels/cwise_op_div.cc

```diff
@@ -23,8 +23,8 @@ REGISTER8(BinaryOp, CPU, "Div", functor::safe_div, uint8, uint16, uint32,
           uint64, int8, int16, int32, int64_t);
 REGISTER8(BinaryOp, CPU, "TruncateDiv", functor::safe_div, uint8, uint16,
           uint32, uint64, int8, int16, int32, int64_t);
-REGISTER4(BinaryOp, CPU, "TruncateDiv", functor::truncate_div_real, Eigen::half,
-          bfloat16, float, double);
+REGISTER6(BinaryOp, CPU, "TruncateDiv", functor::truncate_div_real, Eigen::half,
+          bfloat16, float, double, complex64, complex128);
```

Comment:
Complex types are _not_ supported.  This breaks a bunch of stuff, and as you can see from the functor, is only supported by _real_ values.  Floor/truncation doesn't make much sense for complex inputs.